knowledge-base/grid-rows-text-ellipsis.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Prevent the Grid from wrapping text in multiline
+description: How to prevent the Grid from wrapping text in multiline
+type: how-to
+page_title: Prevent the Grid from wrapping text in multiline
+slug: grid-kb-rows-text-ellipsis
+position: 
+tags: 
+ticketid: 1499434
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
```

Comment:
Consider adding the TreeList product, and maybe linking this KB https://github.com/telerik/blazor-docs/commit/8603366f60cd43822d5018fbdaa014b48392ac16